manage_arkime/commands/create_cluster.py

```diff
@@ -8,16 +8,22 @@
 from cdk_interactions.cdk_client import CdkClient
 import cdk_interactions.cdk_context as context
 import constants as constants
-from core.capacity_planning import get_capture_node_capacity_plan, get_ecs_sys_resource_plan, CaptureNodesPlan, EcsSysResourcePlan, MINIMUM_TRAFFIC
+from core.capacity_planning import (get_capture_node_capacity_plan, get_ecs_sys_resource_plan, get_os_domain_plan, ClusterPlan,
+                                    CaptureNodesPlan, CaptureVpcPlan, EcsSysResourcePlan, OSDomainPlan, MINIMUM_TRAFFIC,
+                                    DEFAULT_SPI_DAYS, DEFAULT_SPI_REPLICAS, DEFAULT_NUM_AZS)
 
 logger = logging.getLogger(__name__)
 
-def cmd_create_cluster(profile: str, region: str, name: str, expected_traffic: float):
+class MustProvideAllParams(Exception):
+    def __init__(self):
+        super().__init__("If you specify one of the optional capacity parameters, you must specify all of them.")
```

Comment:
why? is this because create vs update which we will have later?

If keeping this, 3 changes
1) remove the stack trace, we shouldn't show stack traces when its a command line arg issue, it confuses people
2) the error message should include ALL the ones required OR the ones missing. (Guessing ALL is easier)
3) the "default" print out of help is confusing, since there aren't defaults i guess, or only defaults when none are set?

Reply:
@awick and I talked it through and agreed the path forward is to store the inputs as well and accept the additional complexity.  Will update.